subprojects/delivery/nupokati/src/main/kotlin/com/avito/android/artifactory_backup/ArtifactoryBackupTask.kt

```diff
@@ -28,12 +35,20 @@ public abstract class ArtifactoryBackupTask : DefaultTask() {
     @get:Input
     public abstract val artifactoryUploadPath: Property<String>
 
+    @Deprecated("included in Artifact for schema version >= 3")
     @get:Input
-    public abstract val buildVariant: Property<String>
+    public abstract val buildConfiguration: Property<String>
 
+    @get:Input
+    public abstract val schemaVersion: Property<Long>
+
+    @Deprecated("use artifacts for schema version >= 3")
     @get:InputFiles
     public abstract val files: Property<FileCollection>
 
+    @get:Input
+    public abstract val artifacts: ListProperty<Deployment>
```

Comment:
Why do you rename it to artifacts?

Reply:
to reflect the change: it's not a filecollection anymore.
But i think deployments is a better name, yes?